usl_pipeline/usl_lib/tests/storage/test_metastore.py

```diff
@@ -211,3 +211,99 @@ def test_flood_scenario_config_delete():
             mock.call.collection().document().delete(),
         ]
     )
+
+
+def test_simulation_label_chunk_is_in_test_set_produce_right_split() -> None:
+    """Ensure is_in_test_set produces a 20-80 split."""
+    study_area = metastore.StudyArea(
+        name="name",
+        col_count=2,
+        row_count=5,
+        x_ll_corner=1.0,
+        y_ll_corner=2.0,
+        cell_size=3.0,
+        crs="over there",
+        chunk_size=1,
+        chunk_x_count=2,
+        chunk_y_count=5,
+    )
+
+    # Calculate whether all chunks are in the validation set.
```

Comment:
Nit: Some of the comments still refer to this as the validation set, which might cause some confusion.

Reply:
sorry, removed those too